### PR TITLE
Script for hybrid JSON template generation

### PR DESCRIPTION
**Pull request by jackal1-66**
This PR is highly connected to https://github.com/AliceO2Group/AliceO2/pull/13699

### CODE REVIEW

**Comment by jackal1-66** [comment]
I see there are multiple tests script that are failing, I will try to fix them directly in this commit (most of them are missing the interaction rate flag)

**Comment by sawenzel** [comment]
> I see there are multiple tests script that are failing, I will try to fix them directly in this commit (most of them are missing the interaction rate flag)

I would prefer fixing unrelated issues in a separate PR.

File: MC/bin/o2_hybrid_gen.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
```
Comment (sawenzel) [inline MC/bin/o2_hybrid_gen.py:2]: please add a short documentation what this tools does or achieves (including a short example usage)

**Comment by jackal1-66** [comment]
I will open a new PR from a different branch. It's curious why this one caused every script in the /run/ folder to be retested, considering no pre-existing script was modified.

**Comment by jackal1-66** [comment]
PR is now opened here https://github.com/AliceO2Group/O2DPG/pull/1808

**Pull request merged**